slippy/tile_grid_test.go

```diff
@@ -348,12 +264,13 @@ func TestFromNativeX(t *testing.T) {
 	}
 }
 
-func TestToNativeX(t *testing.T) {
+func TestToNative(t *testing.T) {
 	type tcase struct {
 		x        uint
+		y        uint
 		srid     uint
 		zoom     uint
```

Comment:
The `zoom`, `x`, and `y` here should also be contained into a tile

Reply:
updated